images/ci-kubernetes-e2e-kubeadm/Dockerfile

```diff
@@ -42,6 +42,9 @@ RUN mkdir -p /tmp/terraform/ && \
     ) && \
     rm -rf /tmp/terraform
 
+# Needed by kubetest
+RUN gcloud components update kubectl
```

Comment:
Would this be better here: https://github.com/kubernetes/test-infra/blob/master/jenkins/e2e-image/Dockerfile#L50 ?